deploy/helm/sifnode/values.yaml

```diff
@@ -26,7 +26,7 @@ sifnode:
   env:
     daemonHome: "/root/.sifnoded"
     daemonName: "sifnoded"
-    daemonAllowDownloadBinaries: false
+    daemonAllowDownloadBinaries: true
```

Comment:
How's this work?

Reply:
See: https://github.com/cosmos/cosmos-sdk/tree/master/cosmovisor#auto-download

Been diving into this yesterday. Looks pretty damn interesting.